### PR TITLE
Added scm.inputMinLines configuration

### PR DESCRIPTION
**Pull request by MaxBR97**

### CODE REVIEW

**Comment by MaxBR97** [comment]
@microsoft-github-policy-service agree

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts [hunk 1]
```diff
@@ -262,6 +262,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
 			maximum: 50,
 			default: 10
 		},
+		'scm.inputMinLines': {
+			type: 'number',
+			markdownDescription: localize('inputMinLines', "Controls the minimum number of lines that the input will auto-grow from."),
```
Comment (jstuyck) [inline src/vs/workbench/contrib/scm/browser/scm.contribution.ts:267]: Type on Controls, remove the s.

Comment (jstuyck) [reply src/vs/workbench/contrib/scm/browser/scm.contribution.ts:267]: My bad. You are correct.

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts [hunk 2]
```diff
@@ -262,6 +262,13 @@ Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).regis
...
+			minimum: 1,
+			maximum: 50,
+			default: 3
```
Comment (lszomoru) [inline src/vs/workbench/contrib/scm/browser/scm.contribution.ts:270]: The default value should be `1` as we do not want to change existing behaviour.

**Review by lszomoru — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please adjust the default value of the setting as requested.

File: src/vs/workbench/contrib/scm/browser/scm.contribution.ts (hunk 2 above)
Comment (MaxBR97) [reply src/vs/workbench/contrib/scm/browser/scm.contribution.ts]: Fixed.

**Review by lszomoru — APPROVED** [review]

**Review by bpasero — APPROVED** [review]

**Pull request merged**